ouroboros-consensus/docs/VersioningSchemeDecision.md

```diff
@@ -0,0 +1,34 @@
+# Choosing a Versioning Schemes
+
+This document records our discussions around choosing how to version our packages.
```

Comment:
Once we've made a decision inside Consensus, we can invite Marcin et al to ask whether they agree/can refine?